Build.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/zsh
```

Comment:
can we lowercase the filename?  ie: "build.sh"

Reply:
Yes! Make sense.